### PR TITLE
Not working on dnxcore50 - references problem during compilation of templates

### PR DESCRIPTION
**Pull request by lajones**
Removing support for dnxcore50 until we can fix.

@Eilon @divega @rowanmiller @bricelam

### CODE REVIEW

File: src/EntityFramework.SqlServer.Design/project.json [hunk 1]
```diff
@@ -15,6 +15,6 @@
   "resource": "ReverseEngineering/Templates/*.cshtml",
   "frameworks": {
     "net45": { },
-    "dnxcore50": { }
+    "dnx451": { }
```
Comment (Eilon) [inline src/EntityFramework.SqlServer.Design/project.json]: Hmm so why is `dnx451` being *added* here?

Comment (lajones) [inline src/EntityFramework.SqlServer.Design/project.json]: @bricelam It got removed during the clean-up that was done around all of the changes to dotnet etc. It appears to work even if you just specify net45. Personally I'd rather be explicit about the fact that we support dnx451 as well but if Brice says it's OK not to have that framework in there then that's OK by me too.

Comment (Eilon) [inline src/EntityFramework.SqlServer.Design/project.json]: If it's not needed I don't think I'd put it there... @bricelam thoughts?

Comment (bricelam) [inline src/EntityFramework.SqlServer.Design/project.json]: It's not needed. I think we should avoid cross-compiling to platforms when we don't need too. It just make the nupkg files bigger than they need to be.

File: test/EntityFramework.SqlServer.Design.FunctionalTests/project.json [hunk 2]
```diff
@@ -10,7 +10,6 @@
     "test": "xunit.runner.aspnet"
   },
   "frameworks": {
-    "dnx451": { },
-    "dnxcore50": { }
+    "dnx451": { }
```
Comment (bricelam) [inline test/EntityFramework.SqlServer.Design.FunctionalTests/project.json]: There doesn't seem to be any reason these tests *shouldn't* be able to run on `dnxcore50`. Is there a test fix we can make instead?

File: src/EntityFramework.SqlServer.Design/project.json (hunk 1 above)
Comment (lajones) [inline src/EntityFramework.SqlServer.Design/project.json]: @Eilon @bricelam  OK. I've added another commit - to just use net45. Tests pass either way.

File: src/EntityFramework.SqlServer.Design/project.json [hunk 3]
```diff
@@ -14,7 +14,6 @@
   },
   "resource": "ReverseEngineering/Templates/*.cshtml",
   "frameworks": {
-    "net45": { },
-    "dnxcore50": { }
+    "net45": { }
```
Comment (bricelam) [inline src/EntityFramework.SqlServer.Design/project.json]: Why doesn't this work on dnxcore50? We need to be able to reverse engineer on CoreCLR...

Comment (lajones) [inline src/EntityFramework.SqlServer.Design/project.json]: It's a problem with the references that get passed to the compile step of the templates. The references look correct but the compiler complains that it doesn't have a "BinaryOperation" member that  it's expecting. I'm working on it. But want to get this checked in so we can have a working release branch without muting this test while I figure out what's going wrong.

**Comment by bricelam** [comment]
:shipit: Sorry, I didn't have enough context. Unblocking for now is good.

**Comment by lajones** [comment]
Fixed with commit 1d973a1. Test running successfully on dnx451.

**Pull request closed without merging**